specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json

```diff
@@ -700,6 +731,34 @@
       ],
       "description": "The parameters used to check the availabity of the storage account name."
     },
+    "SKUCapability": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of capability, The capability information in the specified sku, including file encryption, network acls, change notification, etc."
+        },
+        "value": {
```

Comment:
@lmazuel Isn't the `'true'` a json `string` type and `true` as json `boolean` type? I think if service says it's boolean what are the possible reasons for the service to send string 'true'? Doesn't that make swagger contract wrong? I see that point for better SDK experience but not sure we should open gates where swagger contract gets violated? Thought?